src/routes/webhooks/paddle.ts

```diff
@@ -654,9 +696,12 @@ export const processTransactionCompleted = async ({
   }
 
   const { gifter_id } = (event?.data?.customData ?? {}) as PaddleCustomData;
-
   if (gifter_id) {
     await processGiftedPayment({ event });
+  } else {
```

Comment:
Used the else here, otherwise it would also apply on gifters?

Reply:
I mean probably it should apply, let's check with product